### ISSUE TITLE
nco needs libgomp1

### ISSUE THREAD

**Issue by daf** [ioos/conda-recipes#723]
I was building a Docker image with conda and the ioos channel, and installed nco (4.5).  None of the binaries would work, giving this:

```
root@008a7db6d797:/data# ncks
ncks: error while loading shared libraries: libgomp.so.1: cannot open shared object file: No such file or directory
```

Ended up solving it by adding an `libgomp1` line to my list of things to `apt-get install` in the Dockerfile, but is this something that can be avoided/tested/checked so others aren't so confused as I was?

**Comment by ocefpaf** [comment]
We need to document that.  @bekozi already did a nice work of tracking some these down in #676.

I just need to update the wiki... :disappointed:

**Comment by ocefpaf** [comment]
Fixed in #770
